akka-actor/src/main/scala/akka/util/Unsafe.java

```diff
@@ -61,7 +147,7 @@ public static int fastHash(String str) {
         long s1 = 601258;
         int i = 0;
 
-        if (isJavaVersion9Plus) {
+        if (copyUSAsciiStrToBytesAlgorithm == 1) {
```

Comment:
Might be faster to do a switch on `copyUSAsciiStrToBytesAlgorithm` to hint to the optimizer to give it a tableswitch?